src/common/interfaces/workos-options.interface.ts

```diff
@@ -1,8 +1,6 @@
-import { AxiosRequestConfig } from 'axios';
-
 export interface WorkOSOptions {
   apiHostname?: string;
   https?: boolean;
   port?: number;
-  axios?: Omit<AxiosRequestConfig, 'baseURL'>;
+  config?: RequestInit;
```

Comment:
This is a breaking change as we were exposing `axios` specific options through the constructor.